libs/langchain-google-common/src/connection.ts

```diff
@@ -53,11 +53,28 @@ export abstract class GoogleConnection<
 
   async _clientInfoHeaders(): Promise<Record<string, string>> {
     const clientLibraryVersion = await this._clientLibraryVersion();
+    const clientInfo= await this._getclientInfo();
     return {
       "User-Agent": clientLibraryVersion,
+      "Client-Info": clientInfo.toString(),
```

Comment:
I don't think we want `toString()` here. If this is a Record, then the string would be "[object Object]"